lib/Helper/MimeTypeHelper.php

```diff
@@ -64,6 +64,39 @@ private function guessMimeTypeFromStream(mixed $stream): ?string
         $fileInfo = new finfo(FILEINFO_MIME_TYPE);
         $mimeType = $fileInfo->buffer($bytes);
 
+        // Fallback to extension-based guessing
+        if (!$mimeType || $mimeType === 'application/octet-stream') {
+            $mimeTypes = new MimeTypes();
+            $extensions = $mimeTypes->getExtensions($mimeType);
+
+            if (!empty($extensions)) {
```

Comment:
Variable always exists
```suggestion
            if ($extensions) {
```